apps/home/tests.py

```diff
@@ -55,7 +55,7 @@ class TestUrlPathPatterns:
     """Ensure that all URL paths comply with the best practices.
 
     It is possible to face hard to debug issues, when URL paths do not comply with
-    the best practices. For example, when a URL path is missing trailing / Django
+    the best practices. For demo, when a URL path is missing trailing / Django
```

Comment:
I think it's ok as it is

```suggestion
    the best practices. For example, when a URL path is missing trailing / Django
```